packages/KTable/KTable.vue

```diff
@@ -523,6 +523,7 @@ export default defineComponent({
       return (entity, rowData) => {
         const rowListeners = pluckListeners('row:', ctx.listeners)(rowData, 'row')
         const cellListeners = pluckListeners('cell:', ctx.listeners)(entity, 'cell')
+        const ignoredElements = ['button', 'input', 'a']
```

Comment:
is there any other element that could possible exist inside a row that could fire events besides these? just making sure we have everything covered

Reply:
Also added `select`, I can't think of any others and it's suprisingly hard to google for.

Reply:
Not to be the party-pooper, but what if the user wants a click event on a `div` ? or an `icon` (whatever element that would be)

Reply:
Not sure I follow @adamdehaven , these are the elements that will **NOT** trigger row click event.

Reply:
As shown in the docs, just add `@click.stop`.

`<template v-slot:wrapped>
    <div>Row click event? <div @click.stop="buttonHandler"><KBadge appearance="success">Button click</KBadge></div></div>
  </template>`

Reply:
Ohhhh so the list `button, a, input` is a list of "we already know we should ignore these things? 

If yes, that makes more sense 👍🏼